labman/db/process.py

```diff
@@ -2948,12 +2960,15 @@ def _generate_amplicon_sample_sheet(self):
         str
             The illumina-formatted sample sheet
         """
-        fixed_run_name = SequencingProcess._bcl_scrub_name(self.run_name)
-        data = (
-            'Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,'
-            'index,Sample_Project,Description,,\n'
-            '%s,,,,,NNNNNNNNNNNN,,,,,' % fixed_run_name)
-        return self._format_sample_sheet(data)
+        data = ['%sSample_ID,Sample_Name,Sample_Plate,Sample_Well,'
+                'I7_Index_ID,index,Sample_Project,Well_Description,,'
```

Comment:
Could you add a comment on why the Well_Description (rather than just Description) column name was requested, so, institutionally, we don't forget?